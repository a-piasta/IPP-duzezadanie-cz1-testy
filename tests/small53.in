newRoute;67;vkporsqgtapgrg;psjxtclcmehidwzarmpgdkwl
getRouteDescription;67
getRouteDescription;67
repairRoad;firxoevzvwmunpckmowqmyplu;firxoevzvwmunpckmowqmyplu;1
repairRoad;kkbxbzxuswfbn;ppgykvgbrmjjtfuxmp;6
newRoute;654;qbuusenwjpanxrbhw;firxoevzvwmunpckmowqmyplu
repairRoad;psjxtclcmehidwzarmpgdkwl;firxoevzvwmunpckmowqmyplu;7
extendRoute;654;qbuusenwjpanxrbhw
addRoad;gthvpipwdgqfwlehuxhttrhdbmbkj;kkbxbzxuswfbn;1;3
removeRoad;gthvpipwdgqfwlehuxhttrhdbmbkj;kkbxbzxuswfbn
getRouteDescription;67
newRoute;349;yanpgbgakdt;yanpgbgakdt
getRouteDescription;349
getRouteDescription;67
removeRoad;gthvpipwdgqfwlehuxhttrhdbmbkj;kkbxbzxuswfbn
newRoute;115;vkporsqgtapgrg;firxoevzvwmunpckmowqmyplu
extendRoute;115;pvdvdnbznbqzswecusi
getRouteDescription;67
removeRoad;gthvpipwdgqfwlehuxhttrhdbmbkj;kkbxbzxuswfbn
getRouteDescription;654
getRouteDescription;874
repairRoad;qbuusenwjpanxrbhw;kkbxbzxuswfbn;-3
removeRoad;gthvpipwdgqfwlehuxhttrhdbmbkj;kkbxbzxuswfbn
repairRoad;kkbxbzxuswfbn;gthvpipwdgqfwlehuxhttrhdbmbkj;-15
getRouteDescription;133
getRouteDescription;115
addRoad;gthvpipwdgqfwlehuxhttrhdbmbkj;gthvpipwdgqfwlehuxhttrhdbmbkj;19;3
newRoute;791;yanpgbgakdt;psjxtclcmehidwzarmpgdkwl
newRoute;245;firxoevzvwmunpckmowqmyplu;kkbxbzxuswfbn
newRoute;250;kkbxbzxuswfbn;kkbxbzxuswfbn
removeRoad;gthvpipwdgqfwlehuxhttrhdbmbkj;kkbxbzxuswfbn
extendRoute;349;vkporsqgtapgrg
extendRoute;115;vkporsqgtapgrg
repairRoad;qbuusenwjpanxrbhw;yanpgbgakdt;12
repairRoad;vkporsqgtapgrg;firxoevzvwmunpckmowqmyplu;6
getRouteDescription;791
newRoute;645;kkbxbzxuswfbn;gthvpipwdgqfwlehuxhttrhdbmbkj
repairRoad;vkporsqgtapgrg;yanpgbgakdt;-9
getRouteDescription;519
repairRoad;psjxtclcmehidwzarmpgdkwl;psjxtclcmehidwzarmpgdkwl;7
extendRoute;245;qbuusenwjpanxrbhw
repairRoad;qbuusenwjpanxrbhw;qbuusenwjpanxrbhw;-11
repairRoad;pvdvdnbznbqzswecusi;gthvpipwdgqfwlehuxhttrhdbmbkj;-12
extendRoute;645;ppgykvgbrmjjtfuxmp
getRouteDescription;115
addRoad;yanpgbgakdt;ppgykvgbrmjjtfuxmp;10;2
addRoad;pvdvdnbznbqzswecusi;firxoevzvwmunpckmowqmyplu;12;-1
addRoad;qbuusenwjpanxrbhw;psjxtclcmehidwzarmpgdkwl;3;-15
extendRoute;67;icnvbsftrtzotxkhryztknkidyiuj
addRoad;ppgykvgbrmjjtfuxmp;vkporsqgtapgrg;23;7
removeRoad;ppgykvgbrmjjtfuxmp;vkporsqgtapgrg
extendRoute;349;ppgykvgbrmjjtfuxmp
repairRoad;kkbxbzxuswfbn;kkbxbzxuswfbn;-9
removeRoad;psjxtclcmehidwzarmpgdkwl;qbuusenwjpanxrbhw
removeRoad;ppgykvgbrmjjtfuxmp;vkporsqgtapgrg
extendRoute;791;psjxtclcmehidwzarmpgdkwl
