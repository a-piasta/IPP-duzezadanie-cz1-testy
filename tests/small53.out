1: NIE
2: 
3: 
4: NIE
5: NIE
6: NIE
7: NIE
8: NIE
9: TAK
10: TAK
11: 
12: NIE
13: 
14: 
15: NIE
16: NIE
17: NIE
18: 
19: NIE
20: 
21: 
22: NIE
23: NIE
24: NIE
25: 
26: 
27: NIE
28: NIE
29: NIE
30: NIE
31: NIE
32: NIE
33: NIE
34: NIE
35: NIE
36: 
37: NIE
38: NIE
39: 
40: NIE
41: NIE
42: NIE
43: NIE
44: NIE
45: 
46: TAK
47: TAK
48: TAK
49: NIE
50: TAK
51: TAK
52: NIE
53: NIE
54: TAK
55: NIE
56: NIE
